merkle_drop/cli.py

```diff
@@ -260,3 +263,27 @@ def status(jsonrpc: str, merkle_drop_address: str):
         f"{remaining_seconds}"
         f" ({pendulum.now().add(seconds=remaining_seconds).diff_for_humans(absolute=True)})"
     )
+
+
+@main.command(
+    short_help="Compare the Merkle root of an airdrop file with a deployed contracts one."
+)
+@jsonrpc_option
+@merkle_drop_address_option
+@airdrop_file_argument
+def check_root(jsonrpc: str, merkle_drop_address: str, airdrop_file_name: str):
+    click.echo("Calculate Merkle root by airdrop file...")
+    airdrop_data = load_airdrop_file(airdrop_file_name)
+    merkle_root_file = compute_merkle_root(to_items(airdrop_data)).hex()
+    click.echo(f"Merkle root by airdrop file: '{merkle_root_file}'")
+
+    click.echo("Read Merkle root from contract...")
+    web3 = connect_to_json_rpc(jsonrpc)
+    status = get_merkle_drop_status(web3, merkle_drop_address)
+    merkle_root_contract = status["root"].hex()
+    click.echo(f"Merkle root at contract: '{merkle_root_contract}'")
+
+    if merkle_root_file == merkle_root_contract:
+        click.echo("Both Merkle roots are equal.")
```

Comment:
Can we get that colored in green and red
like here https://github.com/trustlines-protocol/blockchain/blob/master/tools/auction-deploy/auction_deploy/cli.py#L405

Reply:
:heavy_check_mark: